docs/notes.rst

```diff
@@ -1,8 +1,38 @@
-Version 1.9.0
+Version 1.9.1
 =============
 
 This release includes the following features:
 
+* Fast-RTPS-Gen is now an independent project.
+* Header **eClock.h** marked is now deprecated.
+
+It also adds the following bug fixes and improvements:
+
+* Bump to FastCDR v1.0.11.
+* Installation from sources documentation fixed.
+* Fixed assertion on WriterProxy.
+* Fixed potential fall through while parsing Parameters.
+* Removed deprecated guards causing compilation errors in some 32 bits platforms.
+* Improve windows performance by avoiding usage of *_Cnd_timedwait* method.
+* Multicast is sent through *localhost* too.
+* Fixed potential race conditions and deadlocks.
+* Eliminating use of *acceptMsgDirectTo*.
+* Discovery Server framework reconnect/recreate strategy.
+* Removed unused folders.
+* Restored subscriber API.
+* SequenceNumber_t improvements.
```

Comment:
I miss some improvements/fixes:

* dll export of addToCDRMessage()
* Improved XML profile example (not sure this one is important enough to be mentioned though)
* Improved blackbox tests that were failing frequently
* Fixes to latency and throughtput tests
* Added STRICT_REALTIME cmake option

Reply:
> I miss some improvements/fixes:
> 
> * dll export of addToCDRMessage()

This one appears as `Restored subscriber API.`

> * Improved XML profile example (not sure this one is important enough to be mentioned though)

I don't think examples improvements are important enough to be mentioned.

> * Improved blackbox tests that were failing frequently
> * Fixes to latency and throughtput tests

We agreed to don't add tests' related changes.

> * Added STRICT_REALTIME cmake option

That change wasn't present at the time the document was updated.

> Is there a reason why they were not added?

I guess we should add the STRICT_REALTIME change, but, what about the others?